packages/snowpack/src/commands/build.ts

```diff
@@ -61,6 +62,28 @@ export async function command(commandOptions: CommandOptions) {
   const {cwd, config} = commandOptions;
   const messageBus = new EventEmitter();
 
+  // TODO: update this with a more robust / typed log event
+  function logEvt({level, id, msg, args}) {
```

Comment:
Assuming TODO is for this PR, but also `logEvent` or `logPluginEvent` instead of `logEvt`

Reply:
I can change the function name, but TODO wasn’t really for this PR: https://github.com/pikapkg/snowpack/pull/668#discussion_r462612684